platforms/Windows/sdk.wxs

```diff
@@ -417,11 +417,11 @@
 
     <ComponentGroup Id="libcxxshim">
       <Component Directory="WindowsSDK_usr_lib_swift_windows_ARCH" Id="libcxxshim.h">
```

Comment:
This needs to change as well, as while the point where it is installed to was updated, the `Directory` attribute here controls where it is deployed to.

```suggestion
      <Component Directory="WindowsSDK_usr_lib_swift_windows" Id="libcxxshim.h">
```

Reply:
Fixed, thanks!